.github/workflows/build.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: ['1.20', '1.21']
```

Comment:
At least of of the imports requires 1.21 so the github testing suite will error without this change.